<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestOOPCarClass</class>
 <widget class="QMainWindow" name="TestOOPCarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestOOPCar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="carlistWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputlineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showCarspushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListWidget" name="carstSpecificManlistWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="numberlabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestOOPCar.qrc"/>
 </resources>
 <connections/>
</ui>
